components/mpas-framework/src/framework/mpas_stream_manager.F

```diff
@@ -3398,10 +3398,11 @@ subroutine write_stream(manager, stream, blockID, timeLevel, mgLevel, forceWrite
                    !
                    write(err_string,'(a,i4,a)') 'Writing to stream '''//trim(stream % name)//''' would overwrite record ', &
                                                 stream % nRecords, ' in file '''//trim(stream % filename)//''','
-                   STREAM_ERROR_WRITE(trim(err_string))
+                   STREAM_WARNING_WRITE(trim(err_string))
                    write(err_string,'(a)') '    but clobber_mode is set to ''append''.'
-                   STREAM_ERROR_WRITE(trim(err_string))
-                   ierr = MPAS_STREAM_MGR_ERR_CLOBBER_REC
+                   STREAM_WARNING_WRITE(trim(err_string))
+                   ! Make this a warning, not an error
+                   ! ierr = MPAS_STREAM_MGR_ERR_CLOBBER_REC
```

Comment:
I can remove this entirely, rather than commenting it out.  Let me know what you all prefer.

Reply:
Yes please remove it.  Also there's a ton of whitespace changes in this PR.

Reply:
@rljacob, yes, I prefer to include the whitespace changes here.  I have them in a separate commit as I commented above.